sdk/containerregistry/Azure.Containers.ContainerRegistry/tests/SessionRecords/HelloWorld/HandleErrors.json

```diff
@@ -0,0 +1,6 @@
+{
+  "Entries": [],
```

Comment:
Samples should be running in live only mode I believe. Do you know why these were generated?

Reply:
I originally had derived the class from RecordedTestBase rather than SamplesBase.  Glad to know this - thanks!  I'll remove them.